.github/workflows/conda.yml

```
@@ -57,13 +57,13 @@ jobs:
           conda mambabuild continuous_integration/recipes/distributed \
                            --channel dask/label/dev \
                            --no-anaconda-upload \
-                           --output-folder .
+                           --output-folder build
 
           # dask pre-release build
           conda mambabuild continuous_integration/recipes/dask \
                            --channel dask/label/dev \
                            --no-anaconda-upload \
-                           --output-folder .
+                           --output-folder build
```

With the above changes, we would no longer need to build in a specific directory:

```suggestion
                           --output-folder .
```